### DISCUSSION TITLE
SSO GitHub Organizations in iOS 13.X | GitHub mobile app v1.3.4 | Authenticator v6.5.4

### DISCUSSION THREAD

**Discussion by ferantivero** [community/community#2983] [Mobile]
# What 
I’d posted a feature request for SSO support in iOS mobile app. The thing is that I’ve been told that I could have it by login out and in.

Although you’re prompted to authorize your SSO organizations as part of the login in process, the flow (in-browser app or plain safari) seems to be broken for me.

# The Flow

Enter GitHub credentials + Authorize SSO GitHub organization + Enter your organization credentials with 2FA login approval (using Authenticator) - Back to GitHub mobile app.

# Expected

The SSO GitHub organization is authorized.

# Actual (in browser app)

in browser app display an error saying something like: _Safari cannot open the page because the network connection was lost_

![1068A609-A669-4470-9C4C-6534DD71155C](https://user-images.githubusercontent.com/158487/102017679-1b01b080-3d47-11eb-9872-1569ec7958c4.jpeg)

# Actual (plain safari)

GitHub app is displaying nothing (black screen in my case, I guess for my theme of choose).

![6A5FA1C4-9457-4708-A521-03BBE299D2AC](https://user-images.githubusercontent.com/158487/102017621-be05fa80-3d46-11eb-8979-eb13f0d0c93b.png)

![DEAE0ED3-C2E1-424D-91F4-C282E521B24F](https://user-images.githubusercontent.com/158487/102017626-c2321800-3d46-11eb-95fe-fea935db5cd9.png)

**Comment by eliperkins** [comment]
@ferantivero hm, this is interesting! I'd love to get some more information from you about why you're running into the "Safari cannot open the page because the network connection was lost" error, as it sounds like there's something funky going on.

- Are you connected to any VPN or otherwise?
- Are you able to log into GitHub.com in Safari.app (not within the GitHub app)?

Regarding the last two screenshots, logging into the app via a browser session outside of the app is not allowed for security reasons, so this log in will need to be initiated and completed within the app itself, via the iOS's built-in `ASWebAuthenticationSession` API.

If you want to reach out to us privately, feel free to shoot us an email at `mobilefeedback (at) github (dot) com`!

**Reply by ferantivero** [reply]
thanks for replaying @eliperkins, please let me quickly answer your questions here, and if more information is required I could definitely share that via e-Mail:

> Are you connected to any VPN or otherwise?

❌ Nope, the only thing is that my device is enrolled in Intune with `Personal` ownership type.

> Are you able to log into GitHub.com in Safari.app (not within the GitHub app)?

✅ yes, I am. I can login into GitHub.com in Safari.app.

EDIT: I would like to add on top of the _Safari cannot open the page because the network connection was lost_ error, that I can reproduce this when I switch to `Authenticator.app` even without approving the login and immediately returning back to `GitHub.app`

I'm planning to upgrade soon to iOS 14. In case you want some extra details, please drop me a message in the following days.

**Reply by ferantivero** [reply]
alright, so I've just updated to iOS 14.4 + GitHub app 1.5, and I can't reproduce this issue anymore.

**Reply by Enxhi369** [reply]
how can it be fixed buy not uploading the new ios 14.1 >>

**Comment by Enxhi369** [comment]
ow about implementing 3 rd party service like type pf bot uplad app ?

**Comment by nfx** [comment]
are there any updates?... it'll be great for app to support SSO...

**Comment by haris-conga** [comment]
I’m facing this issue where my org has only allowed login via EDGE and even if I set edge as default browser I’m not able to login to GitHub app on ios
